docs/source/changelog.rst

```diff
@@ -10,11 +10,12 @@ Array
 - Add ``atleast_1d``, ``atleast_2d``, and ``atleast_3d`` (:pr:`2760`)
 - Add ``allclose`` (:pr:`2771`)
 - Remove ``random.different_seeds`` from Dask Array API docs (:pr:`2772`)
+- Reimplement ``unique`` to be lazy (:pr:`2775`)
 
 DataFrame
 +++++++++
 
--
+- Force ``unique`` in ``from_bcolz`` to be non-lazy (:pr:`2775`)
```

Comment:
TBH I'm not sure we need this entry since all we have done is maintain the current behavior. Whereas this suggests we have actually changed something.

Reply:
Changed the wording in the last commit.

Reply:
I don't think this needs to be in the changelog, there is no user-facing change here. I guess it depends on changelog ideology, but I wouldn't add it.

Reply:
Yep, that's what I was thinking, but was being cautious. Have dropped this now.